### PR TITLE
SentinelOne V2 fetch-fix

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
Fixed Fetch incidents command

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Additional changes
Fixed some date params in get-threats command 

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by Shellyber** [comment]
@kirbles19 Please review CHANGLOG.md

File: Integrations/SentinelOne-V2/SentinelOne-V2.py [hunk 1]
```diff
@@ -1747,9 +1747,9 @@ def fetch_incidents():
...
 
     current_fetch = last_fetch
-
     incidents = []
-    threats = get_threats_request()
+    last_fetch_date_string = timestamp_to_datestring(last_fetch, '%Y-%m-%dT%H:%M:%S.%fZ')
+    threats = get_threats_request(created_after=last_fetch_date_string)
     for threat in threats:
         # If no fetch threat rank is provided, bring everything, else only fetch above the threshold
         if not FETCH_THREAT_RANK or (FETCH_THREAT_RANK and threat.get('rank') >= FETCH_THREAT_RANK):
```
Comment (Itay4) [inline Integrations/SentinelOne-V2/SentinelOne-V2.py]: need to check `threat.get('rank')` is an `int` so the `>=` won't fail

Comment (Itay4) [inline Integrations/SentinelOne-V2/SentinelOne-V2.py]: `FETCH_THREAT_RANK` will always a value since you set a default, so you can remove this

**Comment by kirbles19** [comment]
@Shellyber Done.

File: Integrations/SentinelOne-V2/SentinelOne-V2.py [hunk 2]
```diff
@@ -18,9 +18,10 @@
 TOKEN = demisto.params().get('token')
 SERVER = demisto.params()['url'][:-1] if (demisto.params()['url'] and demisto.params()['url'].endswith('/')) \
     else demisto.params()['url']
-USE_SSL = not demisto.params().get('insecure', False)
+USE_SSL = not demisto.params().get('insecure', '')
```
Comment (Itay4) [inline Integrations/SentinelOne-V2/SentinelOne-V2.py]: revert this

File: Integrations/SentinelOne-V2/SentinelOne-V2.py [hunk 3]
```diff
@@ -21,6 +21,7 @@
 USE_SSL = not demisto.params().get('insecure', False)
 FETCH_TIME = demisto.params().get('fetch_time', '3 days')
 FETCH_THREAT_RANK = int(demisto.params().get('fetch_threshold', 5))
+FETCH_LIMIT = int(demisto.params().get('fetch_limit'))
```
Comment (Itay4) [inline Integrations/SentinelOne-V2/SentinelOne-V2.py]:
```suggestion
FETCH_LIMIT = int(demisto.params().get('fetch_limit', 10))
```

File: Integrations/SentinelOne-V2/SentinelOne-V2.yml [hunk 4]
```diff
@@ -44,6 +43,11 @@ configuration:
   name: fetch_threat_rank
   required: false
   type: 0
+- defaultvalue: '20'
```
Comment (Itay4) [inline Integrations/SentinelOne-V2/SentinelOne-V2.yml]:
```suggestion
- defaultvalue: '10'
```

**Pull request merged**